requirements.txt

```diff
@@ -1,15 +1,12 @@
 future
 zsl_client
```

Comment:
Yes, there's is a background_task decorator to add a task into job queue, which sends it with help of zsl_client. I think it is not used anywhere and was build as a proof of concept, but it is still there.